mt-kit/core/css/src/components/_buttons.scss

```
@@ -46,7 +46,6 @@ $icon-button-border-radius: 0.25em;
   font-weight: var(--font-weight-normal);
 
   // Effects
-  // transition: all 180ms ease-in-out;
   transition: none 180ms ease-in-out;
   transition-property: background-color, color, border-color;
```

Bare forslag og nok litt preferanse; jeg er fan av i å unngå shorthand syntax i situasjoner som dette, hvor 2 innstillinger gjelder mange properties - litt mer lesbart? ☺️ 
```suggestion
  transition-duration: 180ms;
  transition-timing-function: ease-in-out;
  transition-property: background-color, color, border-color;
```

---

@eirikbacker ja faktisk, blir hakket lettere. Er jo litt forvirrende med en 'none' før man overskriver med en liste av properties etterpå.